app/controllers/responses_controller.rb

```diff
@@ -379,13 +380,18 @@ def enketo
   # The blank form template.
   # Returns a string that's safe to print in a JS script.
   def enketo_form_obj
-    # Terrapin seems to return an ASCII-encoded string, so we must interpret it
-    # as UTF-8 in order for the rest of the page to work for some kinds of forms.
-    command = Terrapin::CommandLine.new("node", ":transformer :xml")
-    command.run(
-      transformer: Rails.root.join("lib/enketo-transformer-service/index.js"),
-      xml: @response.form.odk_xml.download
-    ).force_encoding("utf-8").chomp.html_safe # rubocop:disable Rails/OutputSafety
+    # Save XML to disk instead of trying to pass a massive string
```

Comment:
ah neat way of handling this! is there ever any reason to not write to disk? e.g. if it is a very small string, is it better to just keep it in memory? wondering if there is a threshold, or if it matters at all?